mtags/src/main/scala-3/scala/meta/internal/pc/printer/ShortenedNames.scala

```diff
@@ -136,7 +136,7 @@ class ShortenedNames(val indexedContext: IndexedContext):
               designator.asInstanceOf[Symbol]
             else tpe.termSymbol
           val short = ShortName(sym)
-          if tryShortenName(name) then NoPrefix
+          if tryShortenName(short) then TermRef(NoPrefix, designator)
```

Comment:
```suggestion
          if tryShortenName(short) then TermRef(NoPrefix, sym)
```

Can we do this?
if `designator` is not a Symbol, we will fail at runtime because of this assertion https://github.com/lampepfl/dotty/blob/75d8eea1e943bbd2c605d8411c2d52d69974d6f3/compiler/src/dotty/tools/dotc/core/Types.scala#L2733

Reply:
Changed! That works correctly, thanks!